src/compas_robots/viewer/scene/robotmodelobject.py

```diff
@@ -113,8 +113,8 @@ def add_objects(objects, show_flag):
             for i, obj in enumerate(objects):
                 obj.init()
                 if show_flag:
-                    if i > 0:
-                        parent = objects[i - 1]
+                    # if i > 0:
+                    #     parent = objects[i - 1]
                     self.viewer.scene.add(obj, parent)
                     self.viewer.scene.instance_colors[obj.instance_color.rgb255] = obj
```

Comment:
Even though it's (likely) a temporary fix, we should drop a quick comment in here explaining why this is like it is now